PostController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+
+    public function postAction() {
+
+        //avoid notices warnings
+        !isset($_POST['livechat_license_number']) ? $livechat_license_number =  '': $livechat_license_number = $_POST['livechat_license_number'];
```

Comment:
Use Magento syntax to get the post data , instead of use  php post method.
Also need to correct the variable naming convention.
Eg : 
$postData = $this->getRequest()->getPost();
$livechatLicenseNumber = isset($postData['livechat_license_number']) ? $postData['livechat_license_number'] : "";